### PR TITLE
added: HomePage

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**

### CODE REVIEW

**Review by Namrata-J — APPROVED** [review]

File: index.css [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+.footer--last {
+    position: absolute;
+    bottom: 0;
+}
+.login__details {
+    display: flex;
+    gap: 4rem;
+    align-items: center;
+}
+.card__newArrival:hover {
```
Comment (bhargaviRdy) [inline index.css:191]: try to use either _ or - everywhere, don't combine both, it causes confusion

File: index.html [hunk 2]
```diff
@@ -1,15 +1,233 @@
...
+        
+        <ul class="socials">
+          <li><i class="fab fa-2x fa-twitter"></i></li>
+          <li><i class="fab fa-2x fa-github"></i></li>
+          <li><i class="fab fa-2x fa-linkedin"></i></li>
+      
+        </ul>
+        
+        </div>
+
```
Comment (bhargaviRdy) [inline index.html:222]: format the code in this file i.e., use proper indentation for readability. ex: use prettier

**Review by bhargaviRdy — APPROVED** [review]
The website looks awesome, try to make it responsive, suggested few practices

File: index.css (hunk 1 above)
Comment (SuyashTiwariCoder) [reply index.css:191]: ok

File: index.html (hunk 2 above)
Comment (SuyashTiwariCoder) [reply index.html:222]: ok I will surely do it next time

**Pull request merged**